### PR TITLE
Remove generated resources and ignore them

### PR DESCRIPTION
**Pull request by lu-zero**
As discussed in this week call.

### CODE REVIEW

**Review by mahdanoura — APPROVED** [review]

File: poetry.lock
Comment (egekorkan) [inline poetry.lock:1]: shouldn't we keep the lock file though?

Comment (egekorkan) [reply poetry.lock:1]: I see the reasoning but wouldn't CI tests potentially differ from dev environments?

Comment (lu-zero) [reply poetry.lock:1]: I guess all depends on the tests we'd put here, if you prefer that I can leave it out.

Comment (egekorkan) [reply poetry.lock:1]: Then let's reintroduce it based on tests etc. :) Resolving

**Comment by egekorkan** [comment]
Given that it is approved, it can be merged.

**Pull request merged**